weavetest/cron.go

```diff
@@ -0,0 +1,88 @@
+package weavetest
+
+import (
+	"bytes"
+	"crypto/rand"
+	"sort"
+	"time"
+
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/errors"
+)
+
+// Cron is a in memory implementation of the ticker and scheduler.
+type Cron struct {
```

Comment:
It's used in the `x/gov` package for testing. Not used in a conscious way, the code only needs something that implements the interface and does not fail.
I think this will come in handy for any other functionality that relies on the ticker.